task13.js

```diff
@@ -0,0 +1,25 @@
+// Write a function that compares two arrays for equality (same elements in the same order).
+
+function isArraysEqual(arr1, arr2) {
+  //   ES6
+  if (arr1.length !== arr2.length) {
+    return false;
+  }
+
+  const comparedArrays = arr1.every((value, index) => value === arr2[index]);
+  return comparedArrays;
```

Comment:
U don't need to create separate variable if u a not gonna use it in the future
`return  arr1.every((value, index) => value === arr2[index]);`